doc/source/tutorial/stats/continuous_studentized_range.rst

```diff
@@ -0,0 +1,38 @@
+.. _continuous-studentized_range:
+
+Studentized Range Distribution
```

Comment:
We already fixed [one error](https://github.com/scipy/scipy/pull/13732/files#r626028923), but there is at least one other clear error in the documentation and/or tutorial. Please individually review thoroughly against a reference and let me know on slack what you find.